docs/api/portable-stories.md

```diff
@@ -0,0 +1,430 @@
+
+  await Primary.load();
+  render(<Primary />);
+});
+```
+
+### 3. Render
+
+The story has been prepared and can be rendered. You pass it into the rendering mechanism of your choice (e.g. Testing Library render function, Vue test utils mount function, etc.)
+
+👉 For this, you use the [`composeStories`](#composestories) or [`composeStory`](#composestory) API. The composed Story is a renderable component that can be passed to your rendering mechanism.
+
+### 4. Play (optional)
+
+Once the component is rendered, you execute the play function which will contain interactions and assertions. This only applies for stories that use the [play function](../essentials/interactions.md#play-function-for-interactions).
+
+👉 For this, you use the [`composeStories`](#composestories) or [`composeStory`](#composestory) API. The composed Story will return a `play` method to be called **after** it has rendered.
+
+For example, your story might have a play function that prepares your component's desired state: in this case, clicking on a button to display a modal. In portable stories, the play function does not run automatically – you have to call it yourself. The play function needs a `canvasElement`, which should be passed by you. A `canvasElement` is the HTML element which wraps your component. Each testing utility provides different ways to retrieve such element, but here's how to do it with Testing Library:
+
+```tsx
+// Button.test.tsx
+import { test } from 'vitest';
+import { render } from '@testing-library/react';
+import { composeStory } from '@storybook/react';
+
+import meta, { Primary as PrimaryStory } from './Button.stories';
+
+test('renders and executes the play function', async () => {
+  const Primary = composeStory(PrimaryStory, meta);
+
+  const { container } = render(<Primary />);
+  await Primary.play({ canvasElement: container });
+});
+```
+
+<Callout variant="info">
+
+If your play function contains assertions (e.g. expect calls), if they fail, your tests will fail accordingly.
+
+</Callout>
+
+## Overriding globals
+
+If your stories behave differently based on [globals](../essentials/toolbars-and-globals.md#globals), such as they render text in English or Spanish, and such behavior can be changed via the toolbar items, you can achieve similar functionality in portable stories by overriding project annotations when composing a story:
+
+```tsx
+// Button.test.tsx
+import { test } from 'vitest';
+import { render } from '@testing-library/react';
+import { composeStory } from '@storybook/react';
+
+import meta, { Primary } from './Button.stories';
+
+test('renders in English', async () => {
+  const PrimaryStory = composeStory(
+    Primary,
+    meta,
+    { globals: { locale: 'en' } } // 👈 Project annotations to override the locale
+  );
+
+  render(<PrimaryStory />);
+});
+
+test('renders in Spanish', async () => {
+  const PrimaryStory = composeStory(Primary, meta, { globals: { locale: 'es' } });
```

Comment:
little reminder to myself to check this, I am not sure if it's globals or globalTypes